service/handlers_substore.go

```diff
@@ -0,0 +1,189 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017-2018 Canonical Ltd
```

Comment:
I wonder if setting only 2018 for new files would be better

Reply:
Okay, I can do that